listenbrainz/mbid_mapping/Dockerfile

```diff
@@ -18,9 +18,9 @@ COPY . /code/mapper
 FROM mbid-mapping-base as mbid-mapping-prod
 
 # service start up scripts
-COPY docker/consul-template.conf /etc/consul-template.conf
 RUN mkdir /etc/service/mapper
```

Comment:
While we are at it, we can probably remove this line. The documentation for [COPY](https://docs.docker.com/engine/reference/builder/#copy) says
> If <dest> doesn’t exist, it is created along with all missing directories in its path.